src/explorer.ts

```diff
@@ -91,4 +111,39 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
     constructor(kind: kuberesources.ResourceKind, readonly id: string) {
         this.resourceId = kind.abbreviation + '/' + id;
     }
-}
\ No newline at end of file
+}
+
+class VirtualKubernetesResource implements KubernetesObject, ResourceNode {
+    readonly resourceId: string;
+
+    constructor (readonly id : string, readonly kind : string, readonly metadata?: any) {
+        this.resourceId = kind + "/" + id;
+    }
+
+    getChildren() : vscode.ProviderResult<KubernetesObject[]> {
+        if (this.kind === 'cluster') {
+            return [
+                new KubernetesKind(kuberesources.allKinds.node),
+                new VirtualKubernetesResource('Workloads', 'workload'),
+                new KubernetesKind(kuberesources.allKinds.service)
+            ];
+        } else if (this.kind === 'workload') {
+            return [
+                new KubernetesKind(kuberesources.allKinds.deployment),
+                new KubernetesKind(kuberesources.allKinds.job),
+                new KubernetesKind(kuberesources.allKinds.pod)
+            ];
+        }
+        return [];
+    }
+
+    getTreeItem() : vscode.TreeItem | Thenable<vscode.TreeItem> {
+        const treeItem = new vscode.TreeItem(this.id, vscode.TreeItemCollapsibleState.Collapsed);
+        treeItem.contextValue = null;
+        if (this.kind === 'cluster' && !this.metadata.active) {
+            treeItem.collapsibleState = vscode.TreeItemCollapsibleState.None;
+            treeItem.contextValue = "vsKubernetes.resource.cluster";
```

Comment:
Seems like this is not identifying that it's a cluster but specifically that it's an _inactive_ cluster?  If so, consider using a more self-documenting string.

Reply:
fixed.